lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -361,16 +361,18 @@ def update_sample(db: DatabaseSession,
 
     if sample.encounter_id:
         assert sample.encounter_id == encounter_id, \
-            f"Sample {sample.id} already linked to another encounter {sample.encounter_id}"
-        return 
+            f"Sample already linked to another encounter {sample.encounter_id}"
+        return
+
+    LOG.info(f"Updating sample {sample.id}")
```

Comment:
This is a functional change in a whitespace-only commit. I suspect rebasing `census_tract_as_string` onto `sch_clinical_updates` introduced this without you realizing. One quick way to check that a whitespace-only commit contains only whitespace is to examine the output of `git show --ignore-all-space COMMIT`.

Reply:
Good catch!